drivers/SmartThings/harman-luxury/src/api/apis.lua

```diff
@@ -0,0 +1,238 @@
+local get = require "api.gets"
+local set = require "api.sets"
+local invoke = require "api.invokes"
+
+----------------------------------------------------------
+--- Definitions
+----------------------------------------------------------
+
+--- system paths -----------------------------------------
+
+local UUID_PATH = "settings:/system/memberId"
+local MAC_PATH = "settings:/system/primaryMacAddress"
+local MEMBER_ID_PATH = "settings:/system/memberId"
+local MANUFACTURE_NAME_PATH = "settings:/system/manufacturer"
+local DEVICE_NAME_PATH = "settings:/deviceName"
+local MODEL_NAME_PATH = "settings:/system/modelName"
+local PRODUCT_NAME_PATH = "settings:/system/productName"
+
+--- SmartThings paths ----------------------------------
+local SMARTTHINGS_PATH = "smartthings:"
+local SMARTTHINGS_AUDIO_PATH = "smartthings:audio/"
+local SMARTTHINGS_MEDIA_PATH = "smartthings:media/"
+
+----------------------------------------------------------
+--- APIs
+----------------------------------------------------------
+
+APIs = {}
+
+--- system APIs ------------------------------------------
+
+--- get UUID from Harman Luxury on ip
+---@param ip string
+function APIs.GetUUID(ip)
+    return get.String(ip, UUID_PATH)
+end
+
+--- get MAC address from Harman Luxury on ip
+---@param ip string
+function APIs.GetMAC(ip)
+    return get.String(ip, MAC_PATH)
+end
+
+--- get Member ID from Harman Luxury on ip
+---@param ip string
+function APIs.GetMemberId(ip)
+    return get.String(ip, MEMBER_ID_PATH)
+end
+
+--- get device manufacture name from Harman Luxury on ip
+---@param ip string
+function APIs.GetManufatureName(ip)
```

Comment:
typo: `Manufacturer`

Reply:
Typo remains; `Manufature` should be `Manufacturer`